.github/scripts/overwrites/stateful.sh

```diff
@@ -7,11 +7,13 @@ npm pkg set description="DevOps Notebooks built on Runme, connected for collabor
 npm pkg set homepage="https://stateful.com"
 npm pkg set contributes.configuration[0].properties[runme.app.baseDomain].default="platform.stateful.com"
 npm pkg set contributes.configuration[0].properties[runme.app.platformAuth].default=true --json
-npm pkg set contributes.configuration[0].properties[runme.app.enableShare].default=true --json
-npm pkg set contributes.configuration[0].properties[runme.experiments.escalationButton].default=true --json
 npm pkg set contributes.configuration[0].properties[runme.server.lifecycleIdentity].default=1 --json
 npm pkg set contributes.configuration[0].properties[runme.app.notebookAutoSave].default="yes"
 npm pkg set contributes.configuration[0].properties[runme.app.panel.runme.cloud].default="\"\"" --json
+npm pkg set contributes.configuration[0].properties[runme.features.forceLogin].default=true --json
```

Comment:
as per suggestion below - with feature flagging overwrites should be an exception and not part of the packaging process